src/lib/api-client.js

```diff
@@ -165,3 +165,10 @@ export function longTermCheckout(studentId, itemAddresses, adminCode, longtermDu
         longtermProfessor
     });
 }
+
+export function patchItemDueDate(studentId, address, date){
+    return patch('student/item', {studentId}, {
+        itemAddress: address,
```

Comment:
NACK: Why is `itemAddress` here? By having it in the body, doesn't that mean its possible for the address to change? We don't want changing addresses.